### PR TITLE
fix hdr MagneticField/ParametrizedEngine

### PR DESCRIPTION
**Pull request by mrodozov**
Fix headers related to #15248

### CODE REVIEW

**Comment by mrodozov** [comment]
please test

File: MagneticField/ParametrizedEngine/src/BFit3D_data.h [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #ifndef BFit3D_data_h
 #define BFit3D_data_h
 
+#include "BFit3D.h"
```
Comment (perrotta) [inline MagneticField/ParametrizedEngine/src/BFit3D_data.h]: Please remove the same include from the .cc implementation file

**Comment by mrodozov** [comment]
please test

**Comment by perrotta** [comment]
+1
- Techncal 
- Jenkins tests pass

**Comment by fabiocos** [comment]
+1

**Pull request merged**